integration/testnet/network.go

```diff
@@ -58,13 +58,23 @@ const (
 	AccessNodeAPIPort = "access-api-port"
 	// AccessNodeAPIProxyPort is the name used for the access node API HTTP proxy port.
 	AccessNodeAPIProxyPort = "access-api-http-proxy-port"
+	// UnstakedAccessNodeAPIPort is the name used for the unstaked access node API port.
+	UnstakedAccessNodeAPIPort = "unstaked-access-api-port"
+	// AccessNodeAPIProxyPort is the name used for the unstaked access node API HTTP proxy port.
+	UnstakedAccessNodeAPIProxyPort = "unstaked-access-api-http-proxy-port"
```

Comment:
Why do we need this? Can't we use same access API port on both staked and unstaked nodes?